app.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+from flask import Flask, abort, jsonify, request, url_for
+import random
+import sqlalchemy as db
+
+
+app = Flask(__name__)
+
+engine = db.create_engine('postgresql+psycopg2://HIDDENALPHABET:password@hiddenalphabet-db.ceqxyonbodui.us-east-1.rds.amazonaws.com:5432/HiddenAlphabet')

```

Comment:
Make sure to delete these credentials from RDS, generate new ones. I would use environment variables in the future, or prompt the user for input, the former is more appropriate here though.
```suggestion
engine = db.create_engine('postgresql+psycopg2://'+os.environ['PG_USERNAME']+':'+os.environ['PG_PASSWORD']+'@'+os.environ['PG_HOST']+':5432/HiddenAlphabet')
```